layouts/partials/why.html

```diff
@@ -41,5 +41,6 @@
                     </div>
             </div>
         </div>
+    </div>
```

Comment:
I added a closing div, thought the partial should close itself, but if that's not true let me know

Reply:
oops i think i left that out. Thanks for the catch. When i merge this, it should go live.